.github/workflows/ci.yml

```diff
@@ -36,6 +42,7 @@ jobs:
 
       - name: Install dependencies
         run: |
+          sudo apt-get install libcairo2-dev libgirepository1.0-dev
```

Comment:
This doesn't seem right to be installing these dev dependencies, everything should have been built already?

Reply:
I removed it for now.